src/client/manager/Vatoms.js

```diff
@@ -121,12 +121,12 @@ class Vatoms {
   }
 
   /**
-   * Deletes a vatom from your inventory
-   * @param  {String} vatomID  Id of the vAtom you want to delete
+   * Removes a vatom from your inventory
```

Comment:
Prefer: "Removes the specified vAtom from the current user's inventory." 

The audience is the developer, and the developer is writing the app for the public. So avoid pronouns like "your", rather refer to "the current user".

Reply:
fixed in commit 
8f8e9e7